core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -489,7 +489,9 @@ class KafkaApis(val requestChannel: RequestChannel,
     val authorizedRequestInfo = mutable.ArrayBuffer[(TopicPartition, FetchRequest.PartitionData)]()
 
     for ((topicPartition, partitionData) <- fetchRequest.fetchData.asScala) {
-      if (!authorize(request.session, Read, new Resource(Topic, topicPartition.topic)))
+      val topicResource = new Resource(Topic, topicPartition.topic)
+      if ((fetchRequest.isFromFollower() && !authorize(request.session, ClusterAction, topicResource)) ||  
```

Comment:
@edoardocomar Is there any benefit from adding `ClusterAction` ACL to topics rather than just use the existing `ClusterAction` for the cluster resource? Since we don't need fine-grained topic-level ACL for this and brokers already need `ClusterAction` access for cluster, wouldn't that work? 
```
if (fetchRequest.isFromFollower && !authorize(request.session, ClusterAction, Resource.ClusterResource))
```

Reply:
Thanks @rajinisivaram 
the ClusterAction on the Cluster resource essential is reserved fro broker-to-broker requests,
this would actually simplify the changes.